Spanner/src/Transaction.php

```diff
@@ -615,6 +648,19 @@ public function commit(array $options = [])
             throw new \BadMethodCallException('The transaction cannot be committed because it is not active');
         }
 
+        if (is_array($this->transactionId) and isset($this->transactionId['begin'])) {
+            unset($this->options['begin']);
+            $res = $this->operation->transaction($this->session, $this->options+[
+                'fromTransaction' => true
+            ]);
+            if (is_array($res) and isset($res['id'])) {
```

Comment:
we do not use `and`, we use `&&`
```suggestion
            if (is_array($res) && isset($res['id'])) {
```

Reply:
I see the `and` has low precedence. Will change this. thank you!

Reply:
Just a reminder that this is done several places throughout the PR, so please update all of them